README.md

```diff
@@ -152,9 +160,10 @@ Attributes:
 | **v_frame**                     | enum [0, 1, 2]                              | V frame offset from v_alignment_start.                                                                                                                                                                                                                                                                                                                                                           |
 | **j_frame**                     | enum [0, 1, 2]                              | J frame offset from j_alignment_start.                                                                                                                                                                                                                                                                                                                                                           |
-| sequence_alignment          | string                                      | Portion of query sequence spanning V-J alignment, reverse complemented if needed.                                                                                                                                                                                                                                                                                                                |
-| germline_alignment          | string                                      | Assembled, aligned, full-length inferred germline sequence spanning the same region as the sequence_alignment field (V-J region). If necessary, a linker composed of N (unknown nucleotide) is added between V and J fragments.                                                                                                                                                                  |
+| sequence_alignment          | string                                      | Gapped alignment of query sequence spanning V-J segmend aligned to germline, reverse complemented if needed.                                                                                                                                                                                                                                                                                                                |
+| germline_alignment          | string                                      | Aligned germline sequence spanning the same region as the sequence_alignment field (V(D)J region). Segments between matched germlines are gapped to match query sequence length.                                                                                                                                                                  |
```

Comment:
This is also gapped not only between V, D  and J, isn't it?

Reply:
yes, and the description fits

Reply:
Description mentions the gaps are between gene sequences, but they can also be inside. Just add "Gapped" at the beginning to indicate this can be gapped the same as `sequence_alignment`.

Reply:
updated